### ISSUE TITLE
Thing Properties - Args Thing ID textbox limits to 255.

### ISSUE THREAD

**Issue by bitshifter10** [sirjuddington/SLADE#640]
**SLADE Version**: (3.1.1.5)
**Operating System**: (OSX)

**Issue Details**:
The text box for inputting a Thing ID in the Thing Properties window (in the Args tab) limits input to [0, 255].

**Comment by Gaerzi** [comment]
Hexen or UDMF?

**Comment by bitshifter10** [comment]
Oh. Hexen.

**Comment by bitshifter10** [comment]
In other places the Thing ID isn't limited, e.g. when setting the properties of things.

**Comment by Gaerzi** [comment]
Okay, thing ID is on two bytes in Hexen format, so the technical limit is 65535. Limiting to 255 is an error. However I'd really recommend using UDMF for any *ZDoom mapping.

**Comment by bitshifter10** [comment]
I also noticed this had been a problem in the past and was fixed.
There is a problem in 3.1.1.6 though that renders the 3d view unusable (moves endlessly left and locks looking up)...

On Sun, Jan 29, 2017 at 18:06, Gaerzi <notifications@github.com> wrote:
Okay, thing ID is on two bytes in Hexen format, so the technical limit is 65535. Limiting to 255 is an error. However I'd really recommend using UDMF for any *ZDoom mapping.
